### PR TITLE
JS Object

### PR DESCRIPTION
**Pull request by jwheel95**
+Created basic submit functions that are called when clicking "continue"
+updated many html ids

Note: Will have some merge conflicts with Jake's last pull. Also, this is a lot of code for simple tasks, might want to refactor later to shorten things up!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,47 +41,47 @@ <h4 class="mb-3">Who are you?</h4>
...
                     </div>
                   </div>
   
                   <div class="mb-3">
-                    <label for="address">Address</label>
-                    <input type="text" class="form-control" id="workAddress" placeholder="1234 Main St" required="">
-                    <div class="invalid-feedback">Please enter the address.
+                    <label for="signUpAddress">Address</label>
+                    <input type="text" class="form-control" id="signUpAddress" placeholder="1234 Main St" required="">
+                    <div class="invalid-feedback">Please enter the asignUpAdress.
```
Comment (kmausolf) [inline index.html:65]: Shouldn't the printed text here say something along the lines of "Please enter an address"?

**Review by kmausolf — COMMENTED** [review]
This seems like a good start!  One thing you will need to keep in mind is that a user may have multiple entries for school, work history, volunteer experience, etc; these will need to be rendered dynamically with JavaScript.  This can be a bummer with just Vanilla JS, but it's totally doable.  It'll mostly be a matter of using things like document.createElement() and appendChild()/removeChild().

**Comment by kmausolf** [comment]
Depending on whether or not you're allowed to use external JS libraries, you should check out some of these cool form frameworks!

https://form.js.org/

http://www.alpacajs.org/

**Review by ghost — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (jakeaguilar) [reply index.html:65]: yeah that was a typo on my part. Line 65 should say "Please enter your address"
```suggestion
                    <div class="invalid-feedback">Please enter your address
```

**Review by jakeaguilar — COMMENTED** [review]
Good, and since we have most of our ids worked out now, ill integrate the add and delete form buttons i tested out on a dummy repo.

**Pull request merged**